roles/edpm_install_certs/tasks/copy_ca_certs.yaml

```diff
@@ -37,6 +37,6 @@
       ansible.builtin.copy:
         src: "{{ cacert_src_path }}/tls-ca-bundle.pem"
         dest: "{{ cacert_dest_path }}/tls-ca-bundle.pem"
-        mode: '0600'
+        mode: '0644'
```

Comment:
i assume this is required to allow non privadged applications to use the pem file to validate certs. if that is correct then this make sense to me to modify. its not mentioned in the commit message but that is what i would infer form context.

Reply:
yes, in the PR comment "Trust the CA bundle. Make it world readable to allow it to be mounted directly."